frontend/src/pages/display-users/components/DisplaysUserList.js

```diff
@@ -15,8 +15,8 @@ function DisplayUserList() {
     handle("GET", "http://localhost:8000/api/users/", formField, getUsers);
   }, []);
 
-  const toEditUser = () => {
-    navigate("/edit-user-page");
+  const toEditUser = (id, name) => {
+    navigate("/edit-user-page", { state: { id: id, name: name } });
```

Comment:
You can skip with assignment since they're having the same name
```suggestion
    navigate("/edit-user-page", { state: { id, name } });
```